articles/channel.html

```diff
@@ -513,7 +513,8 @@ <h3>Detailed Explanations for Channel Operations</h3>
 	each of them will produce a panic for sending on a closed channel.
 	This is the reason why we should avoid concurrent send and close operations on the same channel.
 	In fact, when the data race detector option (<code>-race</code>) is enabled,
-    the standard Go compiler might be very possible to report data racing for concurrent send and close operation cases.
+  a <code>go test -race mypkg </code> might be able to detect data racing for concurrent send and close operation cases,

```

Comment:
The "-race" is a general option for "go test/run/build/install/...", not only for "go test". 
And it is not a runtime warning, it is a panic making the program crash. Is the following description better?

	In fact, when <a href="https://golang.org/doc/articles/race_detector.html">the data race detector compiler option</a> (<code>-race</code>) is enabled,
	concurrent send and close operation cases might be detected at run time and a runtime panic will be thrown.

Reply:
I agree, that's better. I have one modification to suggest: we call this a "go command option" rather than a compiler option. I think while there may be compilation happening under the hood most of the time, you can use this with `test` `build`, `run` and `install`, and people may only think of `go build` as "compile".